src/main/java/Duke/entities/Todo.java

```diff
@@ -0,0 +1,39 @@
+package Duke.entities;
```

Comment:
Also I like how you packaged the different entities, it makes your repo look more organised.